pkg/reconciler/source/resources/build.go

```diff
@@ -106,10 +128,24 @@ func makeBuildpackBuild(source *v1alpha1.Source) (*build.Build, error) {
 						Value: source.Spec.BuildpackBuild.Stack,
 					},
 				},
-				Env: source.Spec.BuildpackBuild.Env,
+				Env: env,
 			},
 		},
-	}, nil
+	}, makeSecret(source), nil
+}
+
+func makeSecret(source *v1alpha1.Source) *corev1.Secret {
+	m := map[string][]byte{}
+
+	// TODO: Support source.Env.ValueFrom
```

Comment:
Is there a Github issue for this?

Reply:
Alternately, we could declare in source that we don't support ValueFrom

Reply:
I created an issue here. We should discuss it further there: https://github.com/google/kf/issues/821